src/odemis/gui/comp/overlay/cryo_feature.py

```diff
@@ -54,13 +56,21 @@ def __init__(self, cnvs, tab_data):
         DragMixin.__init__(self)
         self._mode = MODE_SHOW_FEATURES
         self.tab_data = tab_data
+        self.pm: MeteorTFS2PostureManager = self.tab_data.main.posture_manager
+        self.view_posture = self.tab_data.view_posture.value
+        self.tab_data.view_posture.subscribe(self._on_view_posture_change, init=True)
+
+        # get the tab based on the view posture
+        self.tab_name = "meteor-fibsem" if self.view_posture == SEM_IMAGING else "cryosecom-localization"
```

Comment:
That shouldn't be needed. There is already the `self.tab_data` which points to all that is needed about "the" tab. 
In the current code, it seems used only at one location, which could use tab_data easily.

Reply:
Does that have access to .main_frame, i couldnt figure it out and didnt seem to work otherwise